### DISCUSSION TITLE
如何实现下面这样的效果? 文本分散对齐.

### DISCUSSION THREAD

**Discussion by xingzuhui** [BYJRK/DotNet-Discussions#196] [WPF 开发]
![image](https://github.com/BYJRK/DotNet-Discussions/assets/33748832/74ccf54b-6af8-454e-988e-be8d5629fb3c)
css  一个justify就可以实现了.
可是wpf怎么实现?  外套viewbox的方法不合适,.

**Comment by BYJRK** [comment]
可以试试这个自定义控件：
https://stackoverflow.com/questions/69957231/c-sharp-wpf-justify-single-lines-of-text-in-textblocks

**Comment by xingzuhui** [comment]
这个也太复杂了.自己做了控件这是.
我觉得ap实现才合适.
